inginious/frontend/pages/lti.py

```diff
@@ -173,9 +173,9 @@ def GET(self, courseid, taskid):
     def POST(self, courseid, taskid):
         (sessionid, loggedin) = self._parse_lti_data(courseid, taskid)
         if loggedin:
-            return redirect(self.app.get_homepath() + "/@{}@/lti/task".format(sessionid))
+            return redirect(self.app.get_homepath() + "/lti/task")
```

Comment:
I'm a bit worried that flask thus creates a session when you reach /lti, then redirects you to /lti/task, that creates another session id?